public/app/features/dashboard/dashgrid/PanelHeader/PanelHeaderTitleItems.tsx

```diff
@@ -41,10 +40,10 @@ export function PanelHeaderTitleItems(props: Props) {
       <Tooltip
         content={data.request?.range ? `Time Range: ${data.request.range.from} to ${data.request.range.to}` : ''}
       >
-        <span className={cx(styles.item, styles.timeshift)}>
+        <PanelChrome.TitleItem className={styles.timeshift}>
           <Icon name="clock-nine" />
           {data.request?.timeInfo}
-        </span>
+        </PanelChrome.TitleItem>
```

Comment:
I don't think this is a good match, but we can keep like this for now. but need to loop in @Ijin08 on how to show timeshift. 

I don't think blue link color is correct, nor this TitleItem that has hover effect. (as timeshift is just text info you cannot interact with). So should be something more like a label / badge, or some none link color (purple?)

Reply:
I added the time range to the content of the tooltip, but I am not sure it is a good idea from the point of view of the user 🤔.

![image](https://user-images.githubusercontent.com/239999/212945023-cbea6e2d-2c4f-4558-8e26-fc26bc32c20d.png)

Reply:
@axelavargas you can keep that (Maybe remove the prefix `TimeRange`) and use the same formating as the tooltip for the regular time picker 
![image](https://user-images.githubusercontent.com/10999/213144228-561457b6-09fc-4ecd-bc95-8c99d8a47de0.png)